dhctl/pkg/config/meta.go

```diff
@@ -514,14 +516,22 @@ func (m *MetaConfig) ParseRegistryData() (map[string]interface{}, error) {
 			return nil, fmt.Errorf("cannot base64 decode docker cfg: %v", err)
 		}
 
-		log.DebugF("parse registry data: dockerCfg after base64 decode = %s\n", bytes)
+		log.DebugF("parse registry data: dockerCfg after base64 decode = %s\n", string(bytes))
```

Comment:
in the internal logging functions uses fmt.Sprintf, so we don't need to do conversion.
```
func simpleFormat(format string, a ...interface{}) string {
	var msg string
	if len(a) > 0 {
		msg = fmt.Sprintf(format, a...)
	} else {
		msg = fmt.Sprintf("%s", format)
	}

	return msg
}
```

Reply:
Oh, Thank you. I removed it.